### DISCUSSION TITLE
Checking for errors in opening or updating PVD files

### DISCUSSION THREAD

**Discussion by NimaGhorab** [Kitware/trame#649] [General] [answered]
Hello all.
I'm repeatedly checking for a PVD file change in some directory.
The problem that I have is that sometimes the dependencies of the PVD file have not yet been prepared and because of this, the file won't properly open or update!
As I'm using both PVDReader and ReloadFiles, Is there any way to check if the PVD file is properly loaded?
Right now, If the file is not properly loaded I get some red errors logged in my console without returning any error code or throwing any exceptions at all!
Is there any workaround for this issue?
Thanks a lot in advance.

**Comment by jourdain** [comment]
That should probably be a question on vtk or paraview discourse depending on which one you use.

**Reply by NimaGhorab** [reply]
Thanks a lot, dear Jourdain.
I asked in the Paraview forum. hope that they can help.
I will post the final answer here if I find any.

**Comment by NimaGhorab** [comment, marked as answer]
So I asked in the Paraview forum and Mathieu Westphal from Kitware answered me that I should check the pipeline using DataInformation API.
